source/includes/cqa-limitations.rst

```diff
@@ -1,8 +1,5 @@
-- You cannot run |CQA| on Atlas 
-  :atlas:`multi-tenant </build-multi-tenant-arch>`
-  configurations.
-- You cannot run |CQA| on 
-  standalone deployments.
+- You cannot run |CQA| on Atlas :ref:`shared clusters <shared-tier-cluster>` and :ref:`serverless instances <atlas-choose-serverless>`
```

Comment:
This line is a bit on the long side.  If you're on vim, highlight part of the line in visual mode then run `gw`.

On VS Code, the onboarding stuff calls for an extension called Rewrap.  Highlight the line and open the command pallet, then run Rewrap.  I'd set it to column 90.  That should automatically wrap to the lines you want.

Or, you could manually break it up.